### PR TITLE
Swap stories one and two

### PR DESCRIPTION
**Pull request by katjam**
Trello card: 
https://trello.com/c/mu7Hbodf/444-e-order-of-stories-top-should-be-about-money-as-phone-will-only-see-that-one

## Description

- Swap order of stories 1 & 2 so first journey in the list refers to money.

@neontribe/sea-map

### CODE REVIEW

File: src/Copy/Text.elm [hunk 1]
```diff
@@ -135,36 +135,36 @@ t key =
...
             "You are not alone"
 
-        -- \u{00A0} is a non-breaking space that prevents a single word wrapping to a new line
         Journey1Teaser ->
-            "He threw my phone on the floor and smashed\u{00A0}it"
+            "He considered my money\u{00A0}his"
 
         Journey1Image ->
-            "/Person_1.svg"
+            "/Person_2.svg"
```
Comment (katjam) [inline src/Copy/Text.elm:142]: @Nikomus Do you think we want to swap the women or the numbers?

Comment (Nikomus) [reply src/Copy/Text.elm:142]: I was going to leave, we don't particularly correspond the illustrations to the stories, I don't think.

**Review by Nikomus — APPROVED** [review]
Good swap. I don't mind about the images but I can re-approve if you want to revert 🤺

**Comment by Nikomus** [comment]
Ah, the people's quote marks used to match their outfits..

**Pull request merged**